scripts/download-sdks.sh

```diff
@@ -30,4 +30,5 @@ for sdk in "${sdks[@]}"; do
     echo "Downloading $sdk SDK to $PWD/$sdk ..."
     rm -rf "./$sdk"
     gh run download $runId -n "$sdk-sdk" -D $sdk
+    chmod -R +x $sdk
```

Comment:
doing a recursive `chmod +x` may delay discovering permission issues with packaging - maybe only change what really needs changing, i.e. the crashpad handler executables?

Reply:
Good point, changed that